.vib/jenkins-agent/goss/vars.yaml

```diff
@@ -0,0 +1,14 @@
+binaries:
+  - gosu
+  - java
+directories:
+  - mode: "0775"
+    paths:
+      - /JENKINS_AGENT_VOLUME_DIR/home
```

Comment:
I think this should be a mistake, it looks like an environment variable that has not been properly rendered.

Reply:
Fixed here https://github.com/bitnami/containers/pull/29988.